docs/statistics.md

```diff
@@ -1,269 +1,425 @@
+| pktRcvBuf               | instantaneous     | packets             | -                    | ✓                      | int       |
+| byteRcvBuf              | instantaneous     | bytes               | -                    | ✓                      | int       |
+| msRcvBuf                | instantaneous     | ms (milliseconds)   | -                    | ✓                      | int       |
+| msRcvTsbPdDelay         | instantaneous     | ms (milliseconds)   | -                    | ✓                      | int       |
+
+
+# Detailed Description
+
+## Accumulated Statistics
+
+### msTimeStamp
+
+The time elapsed since the SRT socket has been created (after successful call to `srt_connect(...)` or `srt_bind(...)` function), in milliseconds. Available both for sender and receiver.
+
+### pktSentTotal
+
+The total number of sent data packets, including retransmitted packets. Available for sender.
+
+### pktRecvTotal
+
+The total number of received packets, including retransmitted packets. Available for receiver.
+
+### pktSndLossTotal
+
+The total number of data packets considered or reported as lost at the sender side. Does not correspond to the packets detected as lost at the receiver side. Available for sender.
+
+A packet is considered lost in two cases: 
+1. Sender receives a loss report from a receiver,
+2. Sender initiates retransmission after not receiving an ACK packet for a certain timeout. Refer to `FASTREXMIT` and `LATEREXMIT` algorithms.
+
+### pktRcvLossTotal
 
-## msTimeStamp
+TODO: 
 
-Time elapsed since the SRT socket was started (after successful call to `srt_connect(...)` or `srt_bind(...)`), in milliseconds.
+- Update the description.
+- Think whether we are going to split lost and reordered packets here and provide user a formula how to get lost only packets if not
 
-## pktSentTotal
+The total number of data packets detected as lost at the receiver side. Available for receiver.
 
-The total number of sent data packets, including retransmitted packets. Applicable for data sender.
+If a packet is received out of order, the gap (sequence discontinuity) is also taken into account and the number of lost packets is increased by the discontinuity size independently of the reordering tolerance value.
 
-## pktRecvTotal
+Lost packets detection is based on the gaps in sequence numbers of the SRT DATA packets and is triggered by a newly received packet. An offset is calculated between sequence numbers of the newly arrived DATA packet and the previously received DATA packet (the packet with the highest sequence number). Receiving older packets does not affect this value. The packets from that gap are considered lost,
+and that number is added to the `pktRcvLossTotal` measurement. In the case where
+the offset is negative, the packet is considered late, meaning that it was either
+already acknowledged or dropped by TSBPD as too late to be delivered. Such late
+packets are ignored.
```

Comment:
That's kinda non-precise. If we state that we calculate the offset between the newly arriving packet and the top receiver sequence, depending on this offset the packet is qualified this way:
* off = 1 --> this is the very next packet, as expected
* off > 1 --> we have missing packets
* off < 1 --> late packet

This "late packet" by this definition is a packet that could be either
* received out-of-order (R=0)
* retransmitted (R=1)

The "belated" packets (there's a separate stat for that - `pktRcvBelated`) are packets that:
* Are behind the current ACK
* Are ahead of current ACK, but the receiver buffer reported that it already has this packet (that can also happen when an out-of-order packet arrived, while it has been already recovered by FEC)